src/_main_/utils/constants.py

```diff
@@ -48,3 +48,6 @@
 ME_INBOUND_EMAIL_ADDRESS = (
     "inbound@massenergize.org"if IS_PROD else os.environ.get('POSTMARK_DEFAULT_INBOUND_EMAIL') if IS_LOCAL else "inbound@massenergize.dev"
 )
+
+
+PUBLIC_EMAIL_DOMAINS=["gmail.com", "yahoo.com", "hotmail.com", "outlook.com", "aol.com"]
```

Comment:
There are other email domains to block as senders, including googlegroups.com (used by one cadmin).  
Can we get feedback to add additional domains on this list, and should it be a separate text file.?